src/components/Table/TableToolbar/TableToolbar.jsx

```diff
@@ -1,11 +1,12 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import IconColumnSelector from '@carbon/icons-react/lib/column/20';
-import IconFilter from '@carbon/icons-react/lib/filter/20';
-import { DataTable, Button } from 'carbon-components-react';
+import IconColumnSelector from '@carbon/icons-react/lib/column/16';
```

Comment:
could we have just bumped all the icons up to 20 rather than scaling up all the 16 px ones?

Reply:
Converted all of the icons to 20px. The search icon was rendered within the tableSearch so that was scaled used scss